spec/index.bs

```diff
@@ -7879,6 +7881,10 @@ GPUQueue includes GPUObjectBase;
                     - {{GPUTextureFormat/"bgra8unorm-srgb"}}
                     - {{GPUTextureFormat/"rgb10a2unorm"}}
                     - {{GPUTextureFormat/"rg8unorm"}}
+                - If |source|.{{GPUImageCopyExternalImage/source}} is an {{HTMLCanvasElement}}:
+                    Its [=canvas context mode=] must be `"2d"`, `"webgl"`, or `"webgl2"`.
```

Comment:
"experimental-webgl"?

Reply:
Not sure whether this still needed.

Reply:
"experimental-webgl" does not appear to be present in the HTML spec:
https://html.spec.whatwg.org/multipage/canvas.html#dom-offscreencanvas-getcontext

But even if it were present in the HTML spec as a valid argument to getContext(), I think it passing "experimental-webgl" would still set the canvas context mode to "webgl". So either way I think we can omit it.